syntax = "proto3";

message PredictionList {
  repeated float values = 1;
}

message MlflowProbs {
  repeated PredictionList pred = 1;
  repeated int32 labels = 2;
  bool eof = 3;
  int32 batch_index = 4;
  string client_id = 5;
}


